AlphaWallet/Transfer/ViewModels/ConfirmTransactionTableViewHeaderViewModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ConfirmTransactionTableViewHeaderViewModel.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 13.07.2020.
+//
+
+import UIKit
+
+struct ConfirmTransactionTableViewHeaderViewModel {
+    let title: String
+    let placeholder: String
+    var details: String = String()
```

Comment:
Looks like the property can be all let (but maybe because it is meant to be more flexible)